inc/connector/namespace.php

```diff
@@ -155,13 +192,28 @@ function pre_reschedule_event( $pre, $event ) {
 
 	if ( is_wp_error( $jobs ) || empty( $jobs ) ) {
 		// The job does not exist.
+		if ( $wp_error ) {
+			if ( ! is_wp_error( $jobs ) ) {
+				$jobs = new WP_Error(
+					'schedule_event_false',
```

Comment:
I would recommend making all error codes unique, if the goal is to reference where the error happened going forward? If so, review duplications in this PR.

Reply:
Error codes do at least have file and line number, again I was following core’s example as best I could but will double check

Reply:
Updated this one - we can use a custom error code and message for Cavalcade in these instances.